quiz.rb

```diff
@@ -14,24 +14,29 @@
 # Take a string and return an array of normalized words.
 # normalized words contain no punctuation and are all lower case
 def normalize_words words
+  words.downcase.split /\W+/
 end
 
 # Question 2
 # Take a string and return the count of normalized words
 def word_count words
+  normalize_words.length
 end
 
 # Question 3
 # Take a string and return and array of the unique normalized words.
 def unique_words words
+  words.normalize_words.uniq
```

Comment:
you can only use dot syntax when youre calling a class method on an object of that class